### PR TITLE
Add system preparation, thumbnail deploy and README

### PR DESCRIPTION
**Pull request by anzoman**
System preparation explains how to setup OpenFaaS and MinIO on your VM, whereas thumbnail deploy shows an example of deployed buckets and image-resize function that is connected to bucket-notification trigger which invokes the function when jpg image is uploaded to a MinIO bucket.

### CODE REVIEW

File: thumbnail-deploy-xopera/docker-image-templates/examples/minio-notification-message-example.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            "sourceIPAddress": "172.16.118.255",
+            "accessKey": "AKIAIOSFODNN7EXAMPLE"
+          },
+          "eventTime": "2019-10-22T09:27:27Z",
+          "awsRegion": "",
+          "eventVersion": "2.0"
+        }
+      ],
+      "Key": "anze/hello.jpg"
+    }
```
Comment (cankarm) [inline thumbnail-deploy-xopera/docker-image-templates/examples/minio-notification-message-example.rst:60]: I see plenty "IPs" and also your name. Is this meant as a template? Should we change them to some general terms?

For the Documentation IPs you can refer here: https://tools.ietf.org/html/rfc5737

File: thumbnail-deploy-xopera/docker-image-templates/image-resize-template/function/handler.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import json
+from PIL import Image
+from .minio_handler import MinIO
+import os
+
+THUMBNAIL_SIZES_PX = [100, 160, 200]
+
+node_ip = "10.10.43.213"
```
Comment (cankarm) [inline thumbnail-deploy-xopera/docker-image-templates/image-resize-template/function/handler.py]: Again IP address.

**Review by cankarm — COMMENTED** [review]
Can you review the hard-coded content and repair it?

File: thumbnail-deploy-xopera/docker-image-templates/image-resize-template/function/handler.py (hunk 2 above)
Comment (anzoman) [reply thumbnail-deploy-xopera/docker-image-templates/image-resize-template/function/handler.py]: I replaced this with documentational IP.

File: thumbnail-deploy-xopera/docker-image-templates/examples/minio-notification-message-example.rst (hunk 1 above)
Comment (anzoman) [reply thumbnail-deploy-xopera/docker-image-templates/examples/minio-notification-message-example.rst:60]: Yes this is just a template. I will change these names and values and will rather use some general terms.

**Pull request merged**